azure-pipelines-cloud-db-integration.yml

```
@@ -0,0 +1,89 @@
+stages:
+  - stage: cloud_db_integration_expectations_cfe
+    pool:
+      vmImage: 'ubuntu-latest'
+
+    jobs:
+      - job: bigquery_expectations_cfe
+        timeoutInMinutes: 0 # Maximize the time that pipelines remain open (6 hours currently)
+
+        variables:
+          python.version: '3.8'
+
+        steps:
+          # delay the execution of the second stages so that we do not hit the rate limit for BigQuery
+          # - bash: sleep 5m
+          #  displayName: Delay for BigQuery rate limit
+
+          - task: UsePythonVersion@0
+            inputs:
+              versionSpec: '$(python.version)'
+            displayName: 'Use Python $(python.version)'
+
+          - bash: python -m pip install --upgrade pip==20.2.4
```

I suggest removing specifying a specific pip version, since that isn't used elsewhere, e.g. at https://github.com/great-expectations/great_expectations/blob/66d9e7e1daa8a538a1929ab87ff4055af9caeefe/azure-pipelines.yml#L471

```suggestion
          - bash: python -m pip install --upgrade pip
```